src/Microsoft.Sbom.Api/Executors/PackagesWalker.cs

```diff
@@ -29,6 +29,8 @@ protected override IEnumerable<ScannedComponent> FilterScannedComponents(ScanRes
         return result
             .ComponentsFound
             .Where(component => !(component.Component is SpdxComponent)) // We exclude detected SBOMs from packages section and reference them as an ExternalReference
+            .GroupBy(component => component.Component.Id)
```

Comment:
Now that both the RustCrateDetector and the RustCli detector are both running we may have duplicates. In this scenario we always want to take the one that came from the RustCli as this one contains Author and Supplier information.